ethexe/rpc/src/apis/tx_pool.rs

```diff
@@ -0,0 +1,103 @@
+// This file is part of Gear.
+//
+// Copyright (C) 2024 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Transaction pool rpc interface.
+
+use crate::{errors, RpcEvent};
+use ethexe_common::tx_pool::{RawTransacton, SignedTransaction, Transaction};
+use gprimitives::{H160, H256};
+use jsonrpsee::{
+    core::{async_trait, RpcResult},
+    proc_macros::rpc,
+};
+use tokio::sync::{mpsc, oneshot};
+
+#[rpc(server)]
+pub trait TransactionPool {
+    #[method(name = "transactionPool_sendMessage")]
+    async fn send_message(
+        &self,
+        program_id: H160,
```

Comment:
I'd consider using here "common::" type just for more friendly target code (js), so nobody's forget across which type we create signature etc

Reply:
moreover, please append todo or impl within the pr cli to submit offchain txs (more likely todo)

Reply:
done